cypress/integration/doiContainer.test.ts

```diff
@@ -4,13 +4,13 @@ describe("DoiContainer", () => {
   })
 
   it("visit 10.70048/q3sn-h087", () => {
-    cy.get('h3.work',  { timeout: 10000 })
+    cy.get('h3.work', { timeout: 30000 })
       .contains('Chandra X-ray Observatory ObsId 1')
       .should('be.visible')
   })
 
   it("export box", () => {
-    cy.get('div#export-xml')
+    cy.get('div#export-xml', { timeout: 30000 })
```

Comment:
Judging by the integration test (also in https://github.com/datacite/akita/pull/42), we still have issues with the GRaphQL API response time. Considering that we also have seen very long response times when retrieving a list of relations, should we add a Task to https://github.com/datacite/lupo/issues/517 (there are only 21 points on that epic, that means there is plenty of time to look at improving response time).

Reply:
I agree that Cypress timeouts need more investigation. But this should be investigated in `akita` first.